articles/image-classifier-keras/index.md

```diff
@@ -0,0 +1,484 @@
+![hero image](/engineering-education/image-classifier-keras/header.jpg)
+
+# How to Build an Image Classifier with Keras
+
+In this article, we will learn how to use a Convolutional Neural Network to build an Image Classifier. We will use Keras with TensorFlow at the backend. Image Classification helps us recognize and identify images. We apply Image Classifiers in fields such as Healthcare, Agriculture, Education, Surveillance, etc.
+
+We will see how we can apply Image Classifiers in Healthcare. Our goal here is to train a CNN model to classify COVID-19 and normal chest X-ray scans of patients.
+
+### Table of Content
+- Importing Libraries and Exploring Dataset
+- Data Visualization
+- Data Preprocessing and Augmentation
+- Build CNN Model
+- Compile and Train the Model
+- Model Evaluation
+- Prediction on New Data
+
+ 
+### Prerequisites
+- Basics of Convolutional Neural Network. I recommend this [article](https://www.section.io/engineering-education/basics-of-convolution-neural-networks/) by Willies Ogola to get started.
+- Python Programming
+- [Colab Notebook](https://colab.research.google.com/)
+
+
+### Importing libraries and exploring the dataset

```

Comment:
Before you begin with the tutorial, ensure you insert a link to your Colab file with the outputs. This way, a person who wants to check your project can do so without having to go through the entire article,